_posts/2022-06-29-omero-5-6-5.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.5
+---
+
+Today we are releasing OMERO.server 5.6.5 which includes:
+
+- a new server configuration allowing to control the pyramidal requirement for floating-point images
+- an upgrade of Bio-Formats to version 6.10.0, see [release announcement]({{ site.baseurl }}/2022/05/31/bio-formats-6-10-0.html).
+- the inclusion of the OMEZarrReader version 0.3.0 for reading OME-NGFF data
+
+Note the Bio-Formats Memoizer cache **will be invalidated** on upgrade from earlier OMERO.server versions.
+
+The loading of OME-NGFF data (version 0.1 to 0.4) is currently only available using the command-line
+interface - see the [limitations](https://omero.readthedocs.io/en/stable/sysadmins/limitations.html) page
```

Comment:
```suggestion
interface - see the [limitations](https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#import-of-ome-ngff) page
```

Reply:
Or is it where we should use the reference that you dutifully introduced in https://github.com/ome/omero-documentation/pull/2248/files#r906087930?

Reply:
Not sure how to technically do it. The  :ref:`minmax_limitation` is used, but from within the ome-docs repo. How to cite from the www repo ?

Reply:
maybe intersphinx https://docs.readthedocs.io/en/stable/guides/intersphinx.html ?

Reply:
I think you can use it just like any heading anchor i.e. https://omero.readthedocs.io/en/stable/sysadmins/limitations.html#ngff-limitations in that case

Reply:
Aha, thank you, updated the commit suggestion @sbesson